src/proposals/20250123/SparkEthereum_20250123.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.25;
+
+import { IAaveV3ConfigEngine as IEngine } from '../../interfaces/IAaveV3ConfigEngine.sol';
+import { IERC20 }                         from 'lib/erc20-helpers/src/interfaces/IERC20.sol';
+import { CCTPForwarder }                  from "xchain-helpers/forwarders/CCTPForwarder.sol";
+
+import { Ethereum }          from 'spark-address-registry/Ethereum.sol';
+
+import { SparkLiquidityLayerHelpers }               from "src/libraries/SparkLiquidityLayerHelpers.sol";
+import { SparkPayloadEthereum, Rates, EngineFlags } from "../../SparkPayloadEthereum.sol";
+
+/**
+ * @title  Jan 23, 2025 Spark Ethereum Proposal
+ * @notice Sparklend: Onboard USDS
+           Spark Liquidity Layer: Onboard Aave Prime USDS, Sparklend USDS and Sparklend USDC. Update CCTP limits
+ * @author Wonderland
+ * Forum:  http://forum.sky.money/t/jan-23-2025-proposed-changes-to-spark-for-upcoming-spell/25825
+ *         http://forum.sky.money/t/jan-23-2025-proposed-changes-to-spark-for-upcoming-spell-2/25837
+ * Vote:   https://vote.makerdao.com/polling/QmRAavx5
+ *         https://vote.makerdao.com/polling/QmY4D1u8
+ *         https://vote.makerdao.com/polling/QmU3Xu4W
+ *         TODO: vote for cctp limits
+ */
+contract SparkEthereum_20250123 is SparkPayloadEthereum {
+
+    address constant public AAVE_PRIME_USDS_ATOKEN = 0x09AA30b182488f769a9824F15E6Ce58591Da4781;
+    address constant public SPARKLEND_USDC_ATOKEN  = 0x377C3bd93f2a2984E1E7bE6A5C22c525eD4A4815;
```

Comment:
Can we use the address from [address registry](https://github.com/marsfoundation/spark-address-registry/blob/master/src/Ethereum.sol#L135) here?